### DISCUSSION TITLE
How to add a new hardware?

### DISCUSSION THREAD

**Discussion by snova-guhae** [axolotl-ai-cloud/axolotl#1902] [Q&A]
Hello! I am looking into adding compatibility with new hardware to axolotl? Namely, I want to add support for the SambaNova RDU. Is there any documentation on how I would add such support or a place where you'd suggest a starting point?

**Comment by NanoCode012** [comment]
Hey, I think a good place would be to check upstream (transformers, accelerate). If it works there, it'll likely be supported here as well.
